src/test/java/seedu/address/TestApp.java

```diff
@@ -65,6 +66,8 @@ protected UserPrefs initPrefs(UserPrefsStorage storage) {
         double x = Screen.getPrimary().getVisualBounds().getMinX();
         double y = Screen.getPrimary().getVisualBounds().getMinY();
         userPrefs.updateLastUsedGuiSetting(new GuiSettings(600.0, 600.0, (int) x, (int) y));
```

Comment:
Can we have variables for these constants?

Reply:
Updated! 👍✔

Reply:
Updated! 👍✔